german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
Für mich ist fern weiter weg als weit, aber ja ich weiß was du meinst. Daher wird ja im Text durch die Angabe der zugehörigen Entfernung in Metern festgelegt was denn nun weiter ist. ;-)

@ghostgroove ja das wäre eine findige Lösung wenn in Reichweite und in Schussweite auch ungünstig ist, weil in Reichweite auch in Schussweite ist. Vor allem fällt in Schussweite raus wenn es sich nicht um Kampfhandlungen handeln und die Distanz für das Erforschen in der Außenwelt erweitert wird.
Es sollten schon neutrale Reichweitenbezeichnungen sein. Und genau da haben wir ein bis zwei Worte, die im Deutschen dafür fehlen. 

Dicht dran, in der Nähe, weit weg, in der Ferne ?

Hab auch alternativ über Nah, Kurz, Weit, Fern nachgedacht aber dann kommt das kurz im sprachlichen Kontext wieder blöd.^^ Wahrscheinlich muss es wirklich etwas umständlicher sein damit es nahtlos in den Sprachgebrauch mit einfließt sein.

Es sei denn es schlägt noch der Lichtblitz ein und es findet sich eine Alternative zu dicht.

Oder ganz alternativ:

Nah, in der Nähe, Weit, Weit entfernt